src/Utils.php

```diff
@@ -317,7 +317,8 @@ public static function streamFor($resource = '', array $options = [])
                  * The 'php://input' is a special stream with quirks and inconsistencies.
                  * We avoid using that stream by reading it into php://temp
                  */
-                if (\stream_get_meta_data($resource)['uri'] === 'php://input') {
+                $metaData = \stream_get_meta_data($resource);
+                if (array_key_exists('uri', $metaData) && $metaData['uri'] === 'php://input') {
```

Comment:
please use `isset`, to be the same as your other PR

Reply:
So... I think we need to call `\stream_get_meta_data` once.

Also I think assign `$metaData` into `if`is not a cool solution.

It is look like good now?